test/WebSites/BasicWebSite/Controllers/HomeController.cs

```diff
@@ -54,6 +55,17 @@ public Task ActionReturningTask()
             return Task.FromResult(true);
         }
 
+        public IActionResult JsonHelperInView()
```

Comment:
add a functional test to one of the classes using `BasicWebSite` that invokes this action.  e.g. add to `BasicTests`.